modules/apps/analytics/analytics-settings-impl/src/main/java/com/liferay/analytics/settings/internal/configuration/AnalyticsConfigurationRegistryImpl.java

```diff
@@ -235,7 +235,13 @@ private void _disable(long companyId) {
 						AnalyticsDXPEntityBatchExporterConstants.
 							DISPATCH_TRIGGER_NAME_ORDER,
 						AnalyticsDXPEntityBatchExporterConstants.
-							DISPATCH_TRIGGER_NAME_PRODUCT
+							DISPATCH_TRIGGER_NAME_PRODUCT,
```

Comment:
@rachaelkoestartyo can you sort?

Reply:
i sorted by the class name AnalyticsDXPEntityBatchExporterConstants, AnalyticsMachineLearningConstants. should we sort by the constant name?

Reply:
Oh I see now :smile:
LGTM
Thanks, Rachael!